src/services/helpers.ts

```diff
@@ -194,12 +194,19 @@ export const getSortedAndFilteredVocabularyItems = (
   vocabularyItems: VocabularyItem[] | null,
   searchString: string
 ): VocabularyItem[] => {
+  const collator = new Intl.Collator('de-De', { sensitivity: 'base', usage: 'sort' })
+
   const normalizedSearchString = normalizeSearchString(searchString)
 
+  const getNouns = (item: { word: string }) => {
```

Comment:
You can simplify the function declaration to `const getNouns = (word: string): string => ...`

Reply:
I have tried this way, but it was giving off error

Reply:
I have tried to write this way, but it gives off an error

Reply:
you can just pass a `word` to this function when you call it  instead of an `item` , since you only need the word(string) and not the entire `item`
Then steffis hint work